pkg/network/openshift_sdn.go

```diff
@@ -48,9 +48,13 @@ func renderOpenShiftSDN(conf *operv1.NetworkSpec, manifestDir string) ([]*uns.Un
 		"metrics-bind-address":    {"0.0.0.0"},
 		"metrics-port":            {"9101"},
 		"healthz-port":            {"10256"},
-		"proxy-mode":              {"iptables"},
+		"proxy-mode":              {"unidling+iptables"},
```

Comment:
This is wrong: we have to apply the value of `c.EnableUnidling` after merging `conf` and `kpcDefaults`. If a 4.1 user redundantly specified `proxy-mode: iptables`, then in 4.1 they would have gotten unidling, but the way this code is currently written here, in 4.2 they would get no unidling. (There should be a test case for this.)

(The simplest way to do this might be to pass `c.EnableUnidling` to `kubeProxyConfiguration()`.)

Reply:
Argh, good point. The "unidling+" is an implementation detail.